github/Team.py

```diff
@@ -234,13 +235,20 @@ def get_repo_permission(self, repo):
         except UnknownObjectException:
             return None
 
+    @deprecated(
+        reason="""
+        Team.set_repo_permission() is deprecated, use
+        Team.update_team_repository() instead.
+        """
+    )
     def set_repo_permission(self, repo, permission):
         """
         :calls: `PUT /teams/:id/repos/:org/:repo <http://developer.github.com/v3/orgs/teams>`_
         :param repo: :class:`github.Repository.Repository`
         :param permission: string
         :rtype: None
         """
+
```

Comment:
This blank line you've added can also be removed.

Reply:
Fixed in https://github.com/PyGithub/PyGithub/pull/1509/commits/f61df4a51357d89c2baa1b074a31294535b3a241